MichaelPachec0/nix-config
# vscode changes

- Rework programs attrset.
- FEAT: vimPlugins add extra plugins.
- FEAT: gpg: rework certain portions.
- chore: formatting on ssh.nix
- FEAT: kanshi: rework.
- FEAT: HM: rework 🐈 options.
- ADD: PKG: git-nvim
- ADD: PKGS: mini.move
- ADD: PKGS: neotest-gtest
- ADD: PKGS: sg.nvim
- ADD: PKG: telescope-docker.nvim
- FEAT: hm: make sure that screenlock wallpaper is committed
- ADD: PKG: telescope-software-licenses.nvim
- FEAT: HM: spicetify: change spotify pkg, plugins.
- FEAT: HM: spicetify: App entry.
- FEAT: HM: spicetify:WIP: try and set dev-tools.
- DEBUG:HM:spicetify check env options in launcher.
- NOTE: HM: spicetify: add extra note for later on.
- gh-actions: create a home-manager check.
- BUGFIX:hm: fix bad merge programs.nix
- FEAT: HM: rework 🐈 options.
- gh-actions: create a home-manager check.
- FEAT: hm: hyprland config change.
- gh-actions: create a home-manager check.
- BUGFIX:hm: fix bad merge programs.nix
- FEAT:hm:programs: firefox update.
- FEAT: HM: vscode: config and packages.
